### ISSUE TITLE
Implement clear option in search

### ISSUE THREAD

**Issue by HamshiniR** [Semantic-Org/Semantic-UI-React#4343]
### Problem description
In search we can have a clear icon, so once we enter we can clear it instead of having a separate button for clearing the text.

### Proposed solution
In search as of now we have a search icon in a input field. It would be good if we have search icon in the left and once the text is entered in search we can show the clear option as well. There is a "clearable" prop in Dropdown so as same we can implement in search field also.

![Capture](https://user-images.githubusercontent.com/94442740/156731166-37804c41-f163-46fa-9d92-a2402079e6ba.PNG)
![Capture-1](https://user-images.githubusercontent.com/94442740/156731168-d5d30b4e-159a-4486-9cca-62622eca8086.PNG)

**Comment by deniapps** [comment]
Yes. I was looking for a solution as well. Looks like the problem is that the input does not allow two icons. The workaround I did is switching the "Search" icon to the "Delete" icon after users enter something. 
**No Input:**
![Rankings-Swim-Standards-search](https://user-images.githubusercontent.com/66892370/156781038-f70a4295-a3d6-41be-aa68-b2b2e2305607.png)
**With Input:**
![Rankings-Swim-Standards](https://user-images.githubusercontent.com/66892370/156781053-87732410-8950-47ba-b534-f2e6822b6a25.png)
You can test this at https://swimstandards.com/rankings **by clicking on "Find Me"**

**Updated on 2/28/2024**
The page mentioned above is no longer available for demonstration because I removed the "Find Me" function. However, I have integrated this feature into the main search functionality of our website. You can test it out by visiting https://swimstandards.com/ and using the search box located at the top right of the page.

@gordielachance, I understand the limitations of the workaround. It suits my needs because I display search results as the user types. However, I'm curious if you've come up with a better solution.

**Comment by gordielachance** [comment]
@deniapps nice, but my form does not have a submit button : search is made when clicking the search icon.
I still would like to be able to show two icons.
